py3-psutil.yaml

```diff
@@ -0,0 +1,48 @@
+package:
+  name: py3-psutil
+  version: 5.9.5
+  epoch: 0
+  description: Cross-platform lib for process and system monitoring in Python.
+  copyright:
+    - license: BSD-3-Clause
+  dependencies:
+    runtime:
+      - python3
+      - linux-headers
+      - python3-dev
+
+environment:
+  contents:
+    repositories:
+      - https://packages.wolfi.dev/os
+    keyring:
+      - https://packages.wolfi.dev/os/wolfi-signing.rsa.pub
```

Comment:
I think we can drop the repos and keyring right?

Reply:
yep, these just added for local testing purposes, thanks for the warning.